spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/BatchNormalization.scala

```diff
@@ -209,21 +246,37 @@ class BatchNormalization[T: ClassTag](
     saveMean.resizeAs(runningMean).zero
     saveStd.resizeAs(runningVar).fill(ev.zero)
 
+    val nChannels = _input.size(2)
```

Comment:
should we requirement the format is NCHW?

Reply:
i's already separated in NCHW related methods, not impact on others